eq-author-api/migrations/updateMetadataValue.js

```diff
@@ -0,0 +1,18 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+
+module.exports = function updateMetadataValue(questionnaire) {
+  const typeMatrix = {
+    Text: "textValue",
+    Date: "dateValue",
+    Region: "regionValue",
+    Language: "languageValue",
+  };
+  questionnaire.metadata.forEach(metadata => {
```

Comment:
This probably be `map`. forEach works but is more awkward.

Reply:
Is map really better than `.forEach` when we're not returning anything and just wanna execute a cb on all elements?

Reply:
You are modifying each of the metadata - so I would say map is what you are doing. The only difference is you filter some out to not make changes to.